warehouse/tasks.py

```diff
@@ -207,7 +207,10 @@ def includeme(config):
             Queue("default", routing_key="task.#"),
             Queue("malware", routing_key="malware.#"),
         ),
-        task_routes={"warehouse.malware.tasks.*": {"queue": "malware"}},
+        task_routes={
+            "warehouse.malware.tasks.*": {"queue": "malware"},
+            "warehouse.oidc.tasks.*": {"queue": "oidc"},
```

Comment:
Do we really anticipate that these tasks will be long running/disruptive enough to require their own queue?

Reply:
I only ask because using SQS as a broker, requires setting up a completely separate SQS queue for each route.

Reply:
Nope, they should in general be very fast. I can remove the queue!